apis/management.cattle.io/v3/alerting_types.go

```diff
@@ -81,13 +81,12 @@ type TargetPod struct {
 }
 
 type TargetEvent struct {
-	Type         string `json:"type,omitempty" norman:"required,options=Normal|Warning,default=Warning"`
+	Type         string `json:"eventType,omitempty" norman:"required,options=Normal|Warning,default=Warning"`
```

Comment:
can we rename the parameter itself? I'd rather keep it consistent with json tag

Reply:
yes, we can. I will have a PR for rancher if we rename the parameter.

Reply:
@alena1108, I made change to it and have a corresponding rancher PR https://github.com/rancher/rancher/pull/12240